src/main/java/service/Service.java

```diff
@@ -0,0 +1,37 @@
+package service;
+
+import static constant.Numbers.MOVE_BOUNDARY_NUMBER;
+import static constant.Numbers.RANDOM_NUMBER_RANGE;
+
+import domain.Car;
+import java.util.List;
+
+public class Service {
+    
+    public static List<String> getWinnerNames(List<Car> cars) {
+        int maxScore = cars.stream()
+            .mapToInt(Car::getScore)
+            .max()
+            .orElseThrow(RuntimeException::new);
+
+        List<String> winnerNames = cars.stream()
+            .filter(car -> car.getScore() == maxScore)
+            .map(Car::getName)
+            .toList();
+
+        return winnerNames;
```

Comment:
```suggestion
        return cars.stream()
            .filter(car -> car.getScore() == maxScore)
            .map(Car::getName)
            .toList();
```
곧바로 return 할 객체를 만들었다면 지역 변수에 담지 않고 바로 반환할 수도 있어요!

Reply:
의미적으로 구분하기 위해서 IDE의 잔소리를 무시하고
꿋꿋하게 변수를 만드는 스타일로 작성을 했던 것 같습니다!
불필요하게 변수를 만든다면 (현재는 큰 문제 없겠지만) 메모리 낭비 문제가 발생할까요?

Reply:
이 부분은 

1. 최대 점수 찾기
2. 최대 점수를 갖는 Car 찾기

두 개의 메서드로 분리해서 처리해도 좋겠다는 생각이 들었습니다!

Reply:
음 낭비라고 하기엔 지역변수 선언이 그정도로 개선해야할만큼 무거운 작업이 아니에요.
로직을 작성할 때 변수 만드는 작업에 대해서는 성능 고려를 안하셔도 됩니다!

Reply:
코드가 깔끔해지기도 하고,
이름을 고민하지 않아도 된다는 장점은 확실히 있겠군요! 👍